Lib/uuid.py

```diff
@@ -118,7 +118,7 @@ class UUID:
                     uuid_generate_time_safe(3).
     """
 
-    __slots__ = ('int', 'is_safe')
+    __slots__ = 'int', 'is_safe', '__weakref__'
```

Comment:
please keep parenthesis, IMHO they help for readability.

Reply:
I agree, have added these.